libsplinter/src/rest_api/mod.rs

```diff
@@ -806,19 +806,18 @@ impl RestApiBuilder {
                                         #[cfg(feature = "oauth-github")]
                                         OAuthConfig::GitHub { .. } => OAuthProvider::Github,
                                     };
-                                    Box::new(OAuthUserStoreSaveTokensOperation::new(
+                                    Box::new(OAuthUserStoreSaveUserInfoOperation::new(
                                         oauth_provider,
-                                        oauth_identity_provider.clone(),
                                         user_store,
                                         oauth_user_store,
                                     ))
                                 }
-                                TokenSaveConfig::NoOp => Box::new(SaveTokensToNull),
+                                TokenSaveConfig::NoOp => Box::new(SaveUserInfoToNull),
```

Comment:
`TokenSaveConfig` should be renamed as well

Reply:
Updated